src/functions/template-tags/date.php

```diff
@@ -290,6 +290,13 @@ function tribe_get_start_date( $event = null, $display_time = true, $date_format
 			return '';
 		}
 
+		static $start_dates = [];
+		$cache_key = "{$event->ID}:{$display_time}:{$date_format}:{$timezone}";
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 10 spaces but found 1 space

<sub>hash: bc043358fa9e2e7ef3694ceb1d9bf3c9</sub>